qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/S3BackupAcceptanceIT.java

```diff
@@ -54,47 +52,31 @@
  * maintain consistency via checkpoint records. Other test suites should be set up for this.
  */
 @Testcontainers
+@ZeebeIntegration
 final class S3BackupAcceptanceIT {
-  private static final Network NETWORK = Network.newNetwork();
-
   private final String bucketName = RandomStringUtils.randomAlphabetic(10).toLowerCase();
 
   @Container
-  private final MinioContainer minio =
-      new MinioContainer().withNetwork(NETWORK).withDomain("minio.local", bucketName);
+  private final MinioContainer minio = new MinioContainer().withDomain("minio.local", bucketName);
+
+  @RegisterExtension
+  @SuppressWarnings("unused")
+  final ContainerLogsDumper logsWatcher = new ContainerLogsDumper(() -> Map.of("minio", minio));
 
-  private final ZeebeCluster cluster =
-      ZeebeCluster.builder()
-          .withImage(ZeebeTestContainerDefaults.defaultTestImage())
-          .withNetwork(NETWORK)
+  // cannot auto start, as we need minio to be started before we can configure the brokers
```

Comment:
:thought_balloon: 

A `dependsOn` that supports arbitrary containers would be super cool :star_struck:

Reply:
It's probably easy enough to do (except maybe pesky edge cases like what if the container was not actually started :smile:)

Reply:
You're welcome to try it ;)